guides/jupyter-notebooks.md

```diff
@@ -50,137 +117,305 @@ notebooks/
 
 - [Title](#title)
+- [Learning Goals](#learning-goals)
 - [Imports](#imports)
 - [Introduction](#introduction)
-- [Loading data](#loading-data)
-- [File information](#file-information)
-- [Sections](#sections) (xN)
-- [Exercises](#exercises) (optional)
-- [Additional resources](#additional-resources) (optional)
-- [About this notebook](#about-this-notebook)
+- [Loading Data](#loading-data)
+- [File Information](#file-information)
+- [Main Content](#main-content) (xN)
+- [Exercises](#exercises) (encouraged)
+- [Additional Resources](#additional-resources) (optional)
+- [About this Notebook](#about-this-notebook)
+- [Citations](#citations)
 - [Footer](#footer)
 
 ### Title
 
-Pick a clear, descriptive title. For titles, use the [following Markdown syntax](https://www.markdownguide.org/basic-syntax/#headings):
+Pick a clear, descriptive title. For titles, use [title case
+capitalization](https://docs.astropy.org/en/latest/development/style-guide.html#capitalization)
+and the following [Markdown
+syntax](https://www.markdownguide.org/basic-syntax/#headings):
+
+```
+# My Clear, Descriptive Title
+
+```
+
+### Learning Goals
+
+Every tutorial should contain three to five explicit learning goals. A learning
+goal should describe what a reader should know or be able to do by the end of
+the tutorial that they didn't know or couldn't do before.
+
+Learning goals can be broken down into:
+
+- Skills: what the reader should be able to do by the end of the tutorial.
+- Knowledge: what the reader should understand by the end of the tutorial.
+- Attitudes: what the reader's opinions about the subject matter will be by the
+  end of the tutorial.
+
+To create your tutorial learning goals, write sentences that identify what
+skill, knowledge, or attitude the reader will gain from your tutorial, and use
+strong action verbs (understand, explain, demonstrate, determine, create,
+access, calculate, analyze).
+
+Template learning goal sentence:
+
+By the end of this tutorial, you will be able to (skill) in order to
+(knowledge).
+
+#### Examples
+
+```
+By the end of this tutorial, you will:
+
+- Understand how to use aperture photometry to turn a series of two-dimensional
+  images into a one-dimensional time series.
+- Be able to determine the most useful aperture for photometry on a *Kepler/K2*
+  target.
+- Create your own light curve for a single quarter/campaign of *Kepler/K2* data.
+
+```
 
 ```
-# My clear, descriptive title
+By the end of this tutorial, you will be able to:
+
+- Understand how NASA's Kepler Mission collected and released light curve data
+  products.
+- Download and plot light curve files from the data archive using `Lightkurve`.
+- Access light curve metadata.
+- Understand the time and brightness units.
+
 ```
+
 ### Imports
 
-Import your dependencies near the top of the notebook, explaining why you're including each one. For example:
+.. suggestion to move Imports to after Introduction
```

Comment:
Agree.  We just got some feedback on some of the JWST notebooks that a context-level introduction at the top is often desirable, so this is coming from multiple directions now.